tests/conftest.py

```diff
@@ -1,8 +1,34 @@
 import tempfile
 from pytest import fixture
 
+from fiboa_cli import convert_utils, util
+
 
 @fixture
 def tmp_file():
     with tempfile.NamedTemporaryFile() as out:
         yield out
+
```

Comment:
Are the changes in the tests related to the class-based approach? If not, I think it would simplify this PR if we'd split the test changes into a separate PR.

Reply:
No, but it really helped me in getting the class-based approach to work. First write a test, then refactor...

Reply:
Pfiew. As our test-suite is still limited I tend to add tests when useful. But I can make a separate pull-requests with most of the current test-additions. I've also added tests for variants, eh years, so they would lead to merge conflicts.

Reply:
See https://github.com/fiboa/cli/pull/141